Modules/_testcapimodule.c

```diff
@@ -2587,6 +2587,117 @@ get_datetime_fromtimestamp(PyObject* self, PyObject *args)
     return rv;
 }
 
+static PyObject *
+test_PyDateTime_GET(PyObject *self, PyObject *args)
+{
+    PyObject *obj = NULL;
+    PyObject *rv;
+    int year, month, day;
+
+    if (!PyArg_ParseTuple(args, "O", &obj)) {
```

Comment:
Since the function takes exactly one argument of type PyObject: you can change the call convention from METH_VARARGS to METH_O.

(Same comment for the other added functions.)